### PR TITLE
Add support for Kafka Schema Registry ACL

### PR DESCRIPTION
**Pull request by juha-aiven**
Add support for Kafka Schema Registry authorization.

Add commands to add, list and delete Schema Registry ACL entries.

Add parameters to service update call to be able to toggle authorization on and off.

### CODE REVIEW

File: aiven/client/client.py [hunk 1]
```diff
@@ -744,7 +762,13 @@ def create_connector_config_based_on_current(
...
         return self.verify(
             self.delete,
-            self.build_path("project", project, "service", service, "acl", acl_id),
+            self.build_path("project", project, "service", service, "kafka", "schema-registry", "acl", acl_id),
+        )
+
+    def delete_service_kafka_schema_registry_acl(self, project: str, service: str, acl_id: str) -> Mapping:
+        return self.verify(
+            self.delete,
+            self.build_path("project", project, "service", service, "schema-registry", "acl", acl_id),
```
Comment (tvainika) [inline aiven/client/client.py:771]: Wrong path, missing "kafka" part

Comment (juha-aiven) [reply aiven/client/client.py]: Yes, fixed now.

**Review by tvainika — APPROVED** [review]
LGTM.

Let's merge this (only) after new APIs have been deployed to production?

**Comment by tvainika** [comment]
> But we need also to make a release to officially make the feature available, until then we are free to change this functionality

A good point. I'll merge this right away, and after these features are generally available in the production, then let's make a release of aiven-client officially.

**Pull request merged**